content/change/institutions/uni.md

```diff
@@ -0,0 +1,11 @@
+title: uni
+
+# How can I start an Open Science Initiative at my own university?
+
+For starters, you might want to find out, if there is a national network of open-science initiatives in your country (e.g. [NOSI](https://osf.io/tbkzh/) in Germany, or [UK-OSWG](https://osf.io/vgt3x/) in the UK) and get in touch. They will most likely be happy to support you in your endeavour!
+
+Other than that, you can turn to these resources
+
+* The newly founded Open Science Community at the University Utrecht documented [why and how](https://docs.google.com/document/d/112B-OCArTI-zdDZ6S67BPVHSn6gPL9GgmluoQ35zG40/edit) they started their local Open Science Initiative:
+
+* There is also this guide on [how to establish an Open Science Interest Group]( https://docs.google.com/document/d/112B-OCArTI-zdDZ6S67BPVHSn6gPL9GgmluoQ35zG40/edit), but sadly we haven't been able track down the author of this piece. If you have any hints, please do let us know!
```

Comment:
This is the same link as above, is that intentional?

Reply:
Argh, no! Sorry and thanks for paying attention...